src/js/utils/general-prop-types.js

```diff
@@ -203,6 +203,7 @@ export const skeletonColorsPropType = PropTypes.shape({
 
 export const skeletonPropType = PropTypes.oneOfType([
   PropTypes.bool,
+  PropTypes.object,
```

Comment:
I think instead of adding `PropTypes.object` if we change `end: PropTypes.end` to `end: PropTypes.string` that should resolve the issue